### PR TITLE
Improve (src/Forms/MultipleReplace.cs) a bit

### PR DESCRIPTION
**Pull request by ivandrofly**
/cc @niksedk

### CODE REVIEW

File: src/Forms/MultipleReplace.cs [hunk 1]
```diff
@@ -186,43 +185,32 @@ private void GeneratePreview()
...
-                                newText = newText.Remove(index, item.FindWhat.Length).Insert(index, item.ReplaceWith);
-                                index = newText.IndexOf(item.FindWhat, index + item.ReplaceWith.Length, StringComparison.OrdinalIgnoreCase);
-                            }
-                            while (index >= 0);
+                            newText = newText.Remove(index, item.FindWhat.Length).Insert(index, item.ReplaceWith);
+                            index = newText.IndexOf(item.FindWhat, index + item.ReplaceWith.Length, StringComparison.OrdinalIgnoreCase);
                         }
                     }
                 }
-                if (hit && newText != p.Text)
```
Comment (ivandrofly) [inline src/Forms/MultipleReplace.cs]: If not hit `newText` won't be equals to `p.Text`

Comment (xylographe) [inline src/Forms/MultipleReplace.cs]: See PR #640

Comment (ivandrofly) [inline src/Forms/MultipleReplace.cs]: > See PR #640

:+1:

File: src/Forms/MultipleReplace.cs [hunk 2]
```diff
@@ -210,15 +209,11 @@ private void GeneratePreview()
                     else
                     {
                         int index = newText.IndexOf(item.FindWhat, StringComparison.OrdinalIgnoreCase);
-                        if (index >= 0)
+                        hit = index >= 0;
```
Comment (ivandrofly) [inline src/Forms/MultipleReplace.cs]: I think this should also fix a bug

**Pull request closed without merging**